docs/source/tutorial/package_a_project.md

```diff
@@ -111,29 +111,19 @@ The resulting `.whl` packages only contain the Python source code of the Kedro p
 
 The project configuration is provided separately in a `tar.gz` file, also inside the `dist` folder. This compressed version of the config files excludes any files inside the `local` directory.
 
-### Package recipients
+### Run a packaged project
 
-Recipients of the `.whl` file need to have Python and `pip` on their machines, but do not need to have Kedro installed.
-
-A recipient can install the project by calling:
+To run a packaged project it must first be installed. To install the project, run the following command:
 
 ```bash
 pip install <path-to-wheel-file>
 ```
 
-An executable, `spaceflights`, is placed in the `bin` subfolder of the Python install folder, so the project can be run as follows:
-
-```bash
-python -m spaceflights
-```
+Once your project is installed, it can be run either using the project's CLI or interactively using Python code.
 
-```{note}
-The recipient will need to add a `conf` subfolder. They also need to add `data` and `logs` if the pipeline loads/saves local data or uses logging.
-Alternatively, they can make use of the ``OmegaConfigLoader`` to run the configuration directly from the compressed .tar.gz configuration file by running
-kedro run --conf-source <path-to-compressed-config>.tar.gz
-```
+To run your project using its CLI, run `<project_name>`. To see a full list of the options that the project's CLI supports, you can run `<project_name> --help`.
```

Comment:
I am a bit confused about what it means. If we have a project name `spaceflights`, does it mean I should run `spaceflights` in the terminal? Does it actually works?

Reply:
It works, in `setup.py` we define the default entry point for the script:

```python
entry_point = (
    "{{ cookiecutter.repo_name }} = {{ cookiecutter.python_package }}.__main__:main"
)
```

This entry point allows the user to run the package after installing it by calling the project's name (`cookiecutter.repo_name`).

Reply:
It might be confusing as I may have the wrong understanding of `package_name` and `project_name`.

Reply:
Thanks for explaining, I wasn't awared that we register an entrypoint automatically.

Reply:
I've looked into this more and decided that `<project_name>` is not right here, it would be inconsistent with how we use `<project_name>` elsewhere in the code base (we map it to `{{ cookiecutter.project_name }}`). I have changed this to `<repo_name>` and added a note to the user what is meant by `repo_name`.

I have modified the precedence of both ways of running a packaged project so that your preferred way (`python -m ...` is now first.